### PR TITLE
Tees-women->Check discount '4 tees for the price of 3' is applied for different(size, color) Tees

### PR DESCRIPTION
**Pull request by YuliyaMihaylovaQA**
https://trello.com/c/Njj1Vlhj/470-tc011016004-sale-tees-women-check-discount-4-tees-for-the-price-of-3-is-applied-for-differentsize-color-tees  also add allure.title for test test_four_tees_discount_is_apply_for_similar_tees

### CODE REVIEW

**Comment by YuliyaMihaylovaQA** [comment]
[TC_011.016.004 | Sale > Tees-women->Check discount "4 tees for the price of 3" is applied for different(size, color) Tees](https://trello.com/c/Njj1Vlhj/470-tc011016004-sale-tees-women-check-discount-4-tees-for-the-price-of-3-is-applied-for-differentsize-color-tees)

File: pages/tees_page.py [hunk 1]
```diff
@@ -13,27 +13,104 @@
...
+
+def check_discount_is_applied_for_four_similar_tees():
+    s(TeesPageLocators.DISCOUNT_SUMM).should(have.text('-$24.00'))
+
+
+def check_discount_is_applied_for_four_different_tees():
+    try:
+        s(TeesPageLocators.DISCOUNT_SUMM).should(be.visible)
+    except Exception as e:
+        print("error discount did not apply, write bug report", e)
```
Comment (aqaestuser) [inline pages/tees_page.py:56]: не уверен что стоит так использовать exception и print

File: tests/test_check_discount_for_four_different_tees_is_applied.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import time
+
+import allure
+
+from pages import tees_page
+
+
+@allure.title("Check four different model of Tees for three discount is applied")
+def test_discount_four_tees_for_different_model_is_applies(browser_management):
```
Comment (aqaestuser) [inline tests/test_check_discount_for_four_different_tees_is_applied.py:9]: browser_management autouse - можно не указывать явно

**Review by aqaestuser — APPROVED** [review]
LGTM

File: tests/test_check_discount_for_four_different_tees_is_applied.py (hunk 2 above)
Comment (YuliyaMihaylovaQA) [reply tests/test_check_discount_for_four_different_tees_is_applied.py:9]: спасибо не знала

**Review by ElenaBarteneva — APPROVED** [review]

**Pull request merged**